### ISSUE TITLE
[BUG] Accessibility - Web App - Buttons must have content

### ISSUE THREAD

**Issue by acovher2** [rokwire/illinois-app#5149]
**Describe the bug**
Buttons must have content. On the Orientation pages using Ainspector tool, I could see an invisible button, that is missing text content, label. A label associated with a form control ensures that information about the form control is spoken by screen readers when it receives focus.
[WCAG 3.3.2 Labels or Instructions](https://www.w3.org/TR/WCAG21/#labels-or-instructions)

**To Reproduce**
1. Go to WEb App prod version using Firefox browser with AInspector installed
2. Navigate through to the Update your Privacy Level screen. 
3. Run AInspector

**Expected behavior**
Buttons should have content. 

**Actual behavior**
There is an invisible button with no content that may be accidentally there. Please advise. 

**Screenshots**

![Image](https://github.com/user-attachments/assets/2284b8f0-2c5d-4d8d-aad5-405e0912de5d)

**Desktop (please complete the following information):**
PC:
•	OS: Windows 11
•	Device: Dell laptop
•	Browser: Chrome 135.0.7049.116/Firefox 138.0.1

**Additional context**

**Comment by acovher2** [comment]
This is seen on the Create a Group Form as well.
